src/common/addlayers/ServerService.js

```diff
@@ -350,7 +350,7 @@ var SERVER_SERVICE_USE_PROXY = true;
         lazy: false
       };
 
-      if (!window.location.search.includes('layer=')) {
+      if (goog.isDefAndNotNull(window.location.search) && !window.location.search.includes('layer=')) {
```

Comment:
`.includes` is not universally implemented. I'd suggest changing this to use `.indexOf() >= 0` as the test instead.  Otherwise, looks good.

Reply:
Thanks @theduckylittle . I've incorporated your feedback.